ansible/playbooks/sap-hana-download-media.yaml

```diff
@@ -23,6 +23,48 @@
         group: root
         mode: 0755
 
+    - name: Retrieve account key
+      ansible.builtin.command: >-
+        az storage account keys list \
+        --account-name {{ az_storage_account_name }} \
+        --query "[?contains(keyName,'{{ az_key_name }}')].value" \
+        -o tsv
+      delegate_to: 127.0.0.1
+      no_log: true
+      run_once: true
+      register: az_account_key
+      when: az_sas_token is not defined or az_sas_token == ""
+
+    - name: "Set expiry"
+      ansible.builtin.command: "date -u +'%Y-%m-%dT%H:%MZ' -d '+20 minutes'"
```

Comment:
I guess 20 minutes is enough in a normal deployment, but what happens when it fails and we need to re-deploy? A new token will be generated?

Reply:
Good point! We could make it configurable. On the other hand token is not needed across all the test or even not across all the deployment. Token just needs to be valid during installer download Ansible task in same playbook

Reply:
Hmmm you are correct. I will increase it and run a final VR, merging automatically if it passes since there are 2 approvals